org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -98,6 +103,7 @@ public void testCoverageResult() {
 				ICounter.FULLY_COVERED);
 		assertLine("explicitExceptionFinally.before", ICounter.FULLY_COVERED);
 		assertLine("explicitExceptionFinally.throw", ICounter.FULLY_COVERED);
+		assertLine("implicitExceptionFinally.finally", ICounter.EMPTY);
```

Comment:
With Eclipse compiler:

    Status in line 127: 		} finally { // $line-implicitExceptionFinally.finally$ expected:<[EMPTY]> but was:<[PARTLY_COVERED]>